dashboard/app/config_linux.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2022 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+// The objective of this file is to collect config parts and routines useful for Linux bugs management,
+// thus reducing the size of the dashboard config file.
+
+// canBeVfsBug determines whether a bug could belong to the VFS subsystem itself.
+func canBeVfsBug(bug *Bug) bool {
+	for _, subsystem := range bug.Tags.Subsystems {
+		if subsystem.Name == "vfs" {
+			return true
+		}
+	}
+	return false
+}
+
+func linuxCallToSubsystem(call string) []string {
+	name := linuxCallToSubsystemMap[call]
+	if name != "" {
+		return []string{name}
+	}
+	return nil
+}
+
+func linuxSubsystemToCc(name string) []string {
+	path := linuxSubsystemToPath[name]
+	if path == "" {
+		return nil
+	}
+	return linuxPathToCc[path]
+}
+
+// These are taken from Linux v6.1. There will be no need to store them here once we
+// begin to automatically parse the MAINTAINERS file.
```

Comment:
We have getMaintainers in pkg/report/linux.go - it works on C files as well as directories. We already support "parsing" the MAINTAINERS file by means of invoking get_maintainers.pl.

Is there a way to to use it here?

Reply:
We ultimately need to move the processing of MAINTAINERS file to the dashboard side -- when the subsystem management is implemented, we'll need to periodically re-query and re-parse the file and re-infer subsystems for existing bugs to keep the information up to date. In the end, email-related logic just belongs better to the dashboard than to the syz-manager, which should be only bothered with the fuzzing itself, so I think a bit later we'll completely remove `get_maintainers.pl` calls from there as well.